### PR TITLE
nginx: clarify special handling for cross

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

**Comment by maxice8** [comment]
1.15 is mainline, we package stable

File: srcpkgs/nginx/template [hunk 1]
```diff
@@ -68,41 +69,31 @@ do_configure() {
 		--with-http_realip_module \
 		--with-http_ssl_module \
 		--with-http_stub_status_module \
-		$(vopt_if geoip --with-http_geoip_module)
+		$(vopt_with geoip http_geoip_module)
```
Comment (maxice8) [inline srcpkgs/nginx/template]: please don't do that and test the change

```
=> nginx-1.15.8_1: running pre-configure hook: 01-override-config ...
=> nginx-1.15.8_1: running pre-configure hook: 02-script-wrapper ...
=> nginx-1.15.8_1: running do_configure ...
./configure: error: invalid option "--without-http_geoip_module"
=> ERROR: nginx-1.15.8_1: do_configure: './configure --prefix=${cfgdir} --conf-path=${cfgdir}/nginx.conf --sbin-path=/usr/bin/nginx --pid-path=/run/nginx.pid --lock-path=/var/lock/nginx.lock --user=nginx --group=nginx --http-log-path=/var/log/nginx/access.log --error-log-path=/var/log/nginx/error.log --http-client-body-temp-path=${tmpdir}/client-body --http-proxy-temp-path=${tmpdir}/proxy --http-fastcgi-temp-path=${tmpdir}/fastcgi --http-scgi-temp-path=${tmpdir}/scgi --http-uwsgi-temp-path=${tmpdir}/uwsgi --with-mail --with-mail_ssl_module --with-pcre-jit --with-file-aio --with-http_gunzip_module --with-http_v2_module --with-http_dav_module --with-http_gzip_static_module --with-http_realip_module --with-http_ssl_module --with-http_stub_status_module $(vopt_with geoip http_geoip_module)' exited with 1
=> ERROR:   in do_configure() at srcpkgs/nginx/template:49
```

Comment (CameronNemo) [reply srcpkgs/nginx/template]: Reverted.

**Comment by CameronNemo** [comment]
This is no longer an update PR.

**Pull request merged**